src/components/Card.tsx

```diff
@@ -0,0 +1,23 @@
+interface ComponentProps {
```

Comment:
p3: 이렇게 표현해도 괜찮을 것 같아요. 
```suggestion
interface Props {
```

Reply:
혹은 CardProps를 그냥 만들고, CardHeader, CardContent에서는 상속받아서 사용해도 괜찮을 것 같네요.